### DISCUSSION TITLE
希望绳网还是不要出软件版了

### DISCUSSION THREAD

**Discussion by clcl-maker** [share121/inter-knot#3342] [其他]
github的绳网上手难度较高，可以过滤一些人，如果成为软件，也希望能提高一下用户进入的门槛，比如要用github账号绑定软件，或者整个实名制，抑或一个手机号只能注册两个绳网账号之类的。希望绳网继续存活下去，成为绝绝们的聚集地。

**Comment by nostalgiatan** [comment]
本来只是脱离浏览器的框架账号是不会变的。

**Comment by Firefly704** [comment]
完全同意。别让那些小鬼来这里污染环境，别把绳网变成下一个贴吧

**Comment by Ziew-Ryan** [comment]
同意，门槛还是要高一点，要有APP的话也希望能够和楼主设想的一样，强制绑定Github账号，至于实名制感觉没多大必要，但是万万不能开放手机号注册，不然门槛一下子就会被拉的很低

**Comment by RaphaelHider** [comment]
个人网站可以要求用户实名制吗？有没有懂法的老哥讲解一下。

**Comment by iuhjk** [comment]
非官方不可能实名的吧

**Comment by Junjun280** [comment]
支持！

**Comment by ID104772508** [comment]
支持！！

**Comment by lllliufeng** [comment]
发帖还得到github，所以其实没变多少

**Comment by owofile** [comment]
支持！，我们需要一个良好的环境氛围讨论

**Comment by Robin-txdy** [comment]
同意，不希望绳网变得乌烟瘴气的

**Comment by nzj2004** [comment]
注册难度要有，但软件版是方便咱们的吧

**Comment by kinich-cute** [comment]
什么时候出个入站考试

**Comment by piracy1026** [comment]
支持！！！

**Comment by linzhiyao123** [comment]
支持楼主，但是软件版如果能不降低门槛又保持现在的干净环境的话还是支持的，毕竟会更方便些<img src="https://img-static.mihoyo.com/communityweb/upload/bf7e7cae6c42a727faa3d5e76b4a4c1c.png" alt="帕姆_比心" width="75" >

**Comment by JT9364** [comment]
说实话，app只是方便我们提高流畅度，但账号绑定还是需要github（虽然github注册只需要一个邮箱），再加上发言强制登录github和app端每次进入都需要登录，算是避免机器人了，入站难度没有变

**Reply by JT9364** [reply]
就是不知道之后share租服务器后会不会变（至少现在没有变
